release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-win-es.html

```diff
@@ -24,8 +24,7 @@ <h2><b>Novedades</b></h2>
   <ul>
     <li>Sincronizaci&oacute;n de enlaces (enlaces simb&oacute;licos y enlaces)</li>
     <li>Protecci&oacute;n a&ntilde;adida para evitar que se sincronicen 2 sincronizaciones dentro de la misma carpeta.</li>
-    <li>Ahora los registros se conservan durante 72 horas en lugar de 24.</li>
-    <li></li>
+    <li>Ahora los registros se conservan durante 7 d�as en lugar de 24.</li>
```

Comment:
Special characters (like `í`) must be HTML encoded. We have a script for that (`encode.sh`)

Reply:
Thanks, I updated it in commit: https://github.com/Infomaniak/desktop-kDrive/pull/84/commits/7b8621dacd74de81e04bace43ff7b04a4398d31a.